e2e/libs/utility/utility.py

```
@@ -334,3 +338,18 @@ def get_name_suffix(*args):
+    size = size.replace(" ", "")
+
+    if size.endswith("GiB"):
+        return int(size[:-3]) * 1024 * 1024 * 1024
+
+    if size.endswith("MiB"):
+        return int(size[:-3]) * 1024 * 1024
+
+    if size.isdigit():
+        return int(size)
+
+    assert False, f"Invalid size format: {size}"
```

Ruff (0.8.0)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)